Add bonjour, calico, kata and quartz files (#137)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/quartz.md

---
name: Quartz
category: Messaging/Comms
description: Quartz is a richly featured, open source job scheduling library that can be integrated within virtually any Java application.
download_url: https://www.quartz-scheduler.org/downloads/
works_on_arm: true
supported_minimum_version:
    version_number: 2.3.1
    release_date: 27/03/2019

optional_info:
    homepage_url: http://www.quartz-scheduler.org/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content:
        official_docs: https://github.com/quartz-scheduler/quartz/blob/main/docs/quick-start-guide.adoc
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: No ARM64 release notes are available. Installation and testing is done using released source code tar.

Author: ```suggestion
    other_info: Linux/ARM64 release notes are not available. Installation and testing are done using released source code tar.
```
